modeller/chapter.md

```diff
@@ -135,18 +141,18 @@ if __name__ == "__main__":
 Before we dive into the `render` function, we should discuss a little bit of linear algebra.
 
 ### Coordinate Space
-For our purposes, a Coordinate Space is an origin point and a set of 3 basis vectors, usually the x, y, and z axes.
+For our purposes, a Coordinate Space is an origin point and a set of 3 basis vectors, usually the $x$, $y$, and $z$ axes.
```

Comment:
I introduced MathJax notation in this chapter -- we're using it in a couple of other chapters.